package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generate-github-markdown-css",
-  "version": "2.2.0",
+  "version": "3.0.0",
```

Comment:
Don't bump the version. It's up to the maintainer to decide which version and when to bump. In this case, I have a script to automatically bump.

Reply:
Gotcha, reverted -- just wanted to make sure the bump to minimum Node requirement didn't get lost as a minor version.